database/N/NEFFEX/Fight Back: The Collection/Fight Back

```diff
@@ -0,0 +1,72 @@
+You're never gonna make it, you're not good enough
+There's a million other people with the same stuff
+You really think you're different? Man you must be kidding
+Think you're gonna hit it, but you just don't get it
+It's impossible, it's not probable
+You're irresponsible, too many obstacles
+You gotta stop it yo, you gotta take it slow
+You can't be a pro, don't waste your time no more
+Who the fuck are you to tell me what to do?
+I don't give a damn if you say you disapprove
+I'm gonna make my move, I'm gonna make it soon
+And I'll do it 'cause it's what I want to fucking do
+'Cause all these opinions and all these positions
+They coming in millions, they're blocking your vision
+But no, you can't listen, that shit is all fiction
+'Cause you hold the power as long as you're driven
+
+You're never gonna make it
+There's no way that you make it
+Yeah maybe you can fake it
+But you're never gonna make it
+
+Are you just gonna take that?
+Make them take it all back
+Don't tell me you believe that
+Are you just gonna take that?
+
+Or will you fucking fight back?
+Or will you fucking fight back?
+
+Yeah
+
+You ain't gonna make it, you ain't never ever gonna break it
+You can never beat 'em man, they're better than you, face it
+Thinking that they give a damn, you're not thinking straight kid
+No, they don't give a damn, you got what I'm saying?
+I'm not fucking playing, I'll give it to you straight, man
+There's too many others and you're not that great, man
+Stop what you're saying, stop what you're making
+Everybody here knows that you're just faking
+Nah, I don't wanna hear it anymore
+I don't wanna hear it anymore
+All these fucking thoughts, they are not what I need anymore
+I'm about to shut the motherfucking door
+On all you poor ass haters with your heads in the clouds
+Talking out loud so proud
+You better shut your goddamn mouths
+Before I do more than speak out, it's about to head south
+
+You're never gonna make it
+There's no way that you make it
+Yeah maybe you can fake it
+But you're never gonna make it
+
+Are you just gonna take that?
+Make them take it all back
+Don't tell me you believe that
+Are you just gonna take that?
+
+Or will you fucking fight back?
+Or will you fucking fight back?
+
+
+__________________________________
+Language          American English
+Name              Fight Back
+Artist            NEFFEX
+Album             Fight Back: The Collection
+Track no          1
+Year              2018
+MusicBrainz ID    cce1780b-3fc0-42bc-8093-f9a99c6e331a
+Original text by  Brandon Horth/Cameron Wales
```

Comment:
Could you please write authors of the original text as: `Brandon Horth / Cameron Wales` or `Brandon Horth,  Cameron Wales` (whichever you think fits better)?